scrapper.py

```diff
@@ -21,21 +34,57 @@ class IncorrectNumberOfArticlesError(Exception):
     """
 
 
+class ResponseIsNotOk(Exception):
+    """
+    Response status code is not ok
+    """
+
+
 class Crawler:
     """
     Crawler implementation
     """
-    def __init__(self, seed_urls, max_articles: int):
-        pass
+
+    def __init__(self, seed_urls, total_max_articles: int):
+        self.seed_urls = seed_urls
+        self.total_max_articles = total_max_articles
+        self.urls = []
+
+    @staticmethod
+    def _changing_the_link(link):
+        if 'http://journals.tsu.ru' not in link:
```

Comment:
looks like some pre-defined domain name that should be defined in a proper place